centreon/src/Core/Security/Vault/Infrastructure/Repository/DbReadVaultConfigurationRepository.php

```diff
@@ -54,26 +54,73 @@ public function findByAddressAndPortAndStorage(
     ): ?VaultConfiguration {
         $this->info('Getting existing vault configuration by address, port and storage');
 
-        $record = [];
         $statement = $this->db->prepare(
             $this->translateDbName(
-                'SELECT * FROM `:db`.`vault_configuration` WHERE `url`=:address AND `port`=:port AND `storage`=:storage'
+                <<<SQL
```

Comment:
php-cs-fixer needs this I think.
```suggestion
                <<<'SQL'
```